docs/project/2024/mohamad_mario.sakka/index.md

```diff
@@ -0,0 +1,100 @@
+# DIY Laptop Cooling Pad
+
+
+**Author**: SAKKA Mohamad-Mario \
+**GitHub Project Link**: <https://github.com/UPB-FILS-MA/project-mmswflow-upb.git>
+
+
+## Description
+
+There are 2 possible of using it:
+* Physically: The fans are controlled through a UI made up of a screen which displays info and buttons with which we increase/decrease the speed of the fans.
+
+* Wirelessly: A user can interact with the microcontroller through a desktop app which sends commands to it through WIFI.
+
+## Motivation
+
+The average temperature around the globe has been on the rise for quite a while, and personally I'm very sensitive to heat, so the first thing that came to my mind was making a fan. After a while I realised that, because I use my laptop for long periods of time doing multiple things including gaming, I wanted also to cool down my laptop since its keyboard gets pretty hot sometimes when playing games, so I've moved on to this idea. I also wanted the cooling pad to be controlled using an app on my laptop through wireless connection, without necessarily having to open it, instead I could use key bindings that I set from the app and I could have the app running in the background.
+
+## Architecture
+
+* The application on the laptop will act as a client, which will connect to the WIFI network hosted on the RP Pico W, and it will send commands to it through TCP to ensure reliable communication between the two. This app will be written in *Python*
+  using *Custom Tkinter & Socket* modules and it will be converted into an executable. The RP PICO W will come with a preset network name & password. 
+
+* The physical UI (buttons & LCD) will be used to start the server, increase/decrease fan speed, power on/off the setup, with all necessary info being displayed on the LCD.
+
+![example](project_architecture.png)  
```

Comment:
I think that this file is missing from the repository.

Reply:
I've uploaded the picture.